.cds/build-manager-dist.pip.yml

```diff
@@ -0,0 +1,18 @@
+version: v1.0
+name: build-manager-dist
+jobs:
+- job: Download
+  steps:
+  - script:
+    - '#!/bin/bash'
```

Comment:
Does the literal style could be used from here? (eg: https://symfony.com/doc/current/components/yaml/yaml_format.html#strings)

Reply:
eg the documentation:

```
Strings containing any of the following characters must be quoted. Although you can use double quotes, for these characters it is more convenient to use single quotes, which avoids having to escape any backslash \:

:, {, }, [, ], ,, &, *, #, ?, |, -, <, >, =, !, %, @, `
```

Reply:
I think the best way is to have external scripts, and our job launch only this script.

We should avoid import our script directly in YAML files. But Step 2.

Here the proposal is to switch to "workflow as code", so it's only an cleaning extract of our remote workflow.

Next step: Make file / NPM script etc etc etc :)

Reply:
Yes I mean/ask if we are able to write the script on multiline or not?

```yaml
|
  echo 'foo'
  cd/mv whatever
  …
```

Reply:
This code is autogenerated by CDS, but i think it's possible :)